Binary files markdown.orig/extensions/__pycache__/wikilinks.cpython-38.pyc and markdown/extensions/__pycache__/wikilinks.cpython-38.pyc differ
diff -urp markdown.orig/extensions/wikilinks.py markdown/extensions/wikilinks.py
--- markdown.orig/extensions/wikilinks.py	2021-04-10 10:13:16.076909532 +0900
+++ markdown/extensions/wikilinks.py	2021-04-10 15:45:15.656921356 +0900
@@ -43,7 +43,7 @@ class WikiLinkExtension(Extension):
         self.md = md
 
         # append to end of inline patterns
-        WIKILINK_RE = r'\[\[([\w0-9_ -]+)\]\]'
+        WIKILINK_RE = r'\[\[([\w0-9\.()_ +-]+)\]\]'
         wikilinkPattern = WikiLinksInlineProcessor(WIKILINK_RE, self.getConfigs())
         wikilinkPattern.md = md
         md.inlinePatterns.register(wikilinkPattern, 'wikilink', 75)
